setup.py

```diff
@@ -71,7 +69,7 @@
     # Test dependencies
     # Execute `pip install .[test]` once and the use `pytest` to run tests
     'test': [
-        'flake8<5', # github.com/ESMValGroup/ESMValCore/issues/1696
+        'flake8<5',  # github.com/ESMValGroup/ESMValCore/issues/1696
```

Comment:
I think we can finally release this one to free too, man - I'll try it in a different PR but you can prob do it here too

Reply:
Good idea, but indeed, let's do it in a new pull request to keep things clean

Reply:
OK lemme do that